pkg/server/event_handler.go

```diff
@@ -355,6 +361,27 @@ func parseGitHubSource(ctx context.Context, data []byte) (*v1alpha1.GitHubSource
 		r.WorkflowSha = ws
 	}
 
+	ra, found := pm.Metadata[MetadataKeyWorkflowRunAttempt]
+	if found {
+		if value, err := strconv.ParseInt(ra, 10, 64); err == nil {
+			r.WorkflowRunAttempt = value
```

Comment:
Could be a follow up - Can we log warns for any value not found or failed to parse?

Reply:
Added #103  for this.